### PR TITLE
[SOC-2408] Bring forward the Social Profile docs to the public site.

### PR DESCRIPTION
**Pull request by wfehr-va**
## [SOC-2408] Bring forward the Social Profile docs to the public site.

This PR takes what we have for the Social Profile endpoint and model on the `products` branch and brings it to master. I've also polished it up and gone through all of the Pre-Release checks: https://vendasta.jira.com/wiki/spaces/API/pages/1651769533/Pre+Release+Checks

[SOC-2408]: https://vendasta.jira.com/browse/SOC-2408?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by richard-rance — COMMENTED** [review]
As discussed, please update the toc.json and changelog.md. You may also write a guide.

**Comment by wfehr-va** [comment]
> As discussed, please update the toc.json and changelog.md. You may also write a guide.

Done, I've added both and fixed up a few minor points. Thanks! :+1:

File: docs/Guides/Social/socialProfiles.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+### Fetching All Connected Social Profiles For A Business
+
+Social Profiles for a single Business Location can be fetched using its ID and an appropriate Authorization token. The data can be used to verify what a business has connected for use in Social Marketing and its current status. Future APIs that are in development will use the IDs obtained from this endpoint to let you access additional functionality.
+
+This example request will enable you to get the first 25 connected Social Profiles for a Business Location, and if there are more, will give you a cursor so you can access the rest. 
+
+<!--
+type: tab
+title: Request
```
Comment (richard-rance) [inline docs/Guides/Social/socialProfiles.md:24]: We should also include a sample response.

File: openapi/social/social.yaml [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+              enum:
+                - facebook
+                - twitter
+                - gmb
+                - linkedinCompany
+                - linkedinUser
+                - instagram
+            status:
+              type: string
+              description: A string enum that describes the current status of this Social Profile.
```
Comment (richard-rance) [inline openapi/social/social.yaml:139]: Could you explain what the status field is without using the word status? Providing examples of when a social profile would be in the disabled or active status may help.

File: openapi/social/social.yaml [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+                      example: AG-12AB5
+                  required:
+                    - type
+                    - id
+          required:
+            - businessLocation
+      required:
+        - type
+      x-lifecycle:
+        Status: Trusted Tester
```
Comment (richard-rance) [inline openapi/social/social.yaml:169]:
```suggestion
        Status: trustedTester
```

File: docs/Guides/Social/socialProfiles.md (hunk 1 above)
Comment (richard-rance) [reply docs/Guides/Social/socialProfiles.md:24]: Users skimming the guide won't see that until they go into operation details. We want them to be able to see what the values should look like without running the command.

**Pull request merged**